stories/useMutationsHandler/usemutationshandler.guideline.mdx

```diff
@@ -0,0 +1,156 @@
+import * as Stories from './usemutationshandler.stories';
+import { Meta, Controls, Story, Canvas, Source } from '@storybook/blocks';
+
+<Meta name="Guideline" of={Stories} />
+
+# useMutationsHandler
+
+useMutationsHandler is a hook to simplify the handling of mutations and the display of feedback to the user.
+It provide a way to define a callback to call on success and a toast to display after the fail or success of a mutation.
+It also allows to define a single toast and callback for multiple related mutations instead of managing each mutation individually.
+
+## Properties
```

Comment:
Does not separate optional and mandatory props, and we said `Props` it's better. I suggest you on chapter `Props` with this kind of description for example : 

`mainMutation`
- Type: MutationConfig<MainMutationType>
- Description: The primary mutation that needs to be executed and monitored.
- Optional (if it is otherwise display nothing)